src/lib/constants.py

```diff
@@ -2,3 +2,7 @@ class SettingFileNames:
     GENERAL_FILE_NAME = "general.json"
     MONITORING_GROUPS_FILE_NAME = "monitoring_groups.json"
     RECIPIENTS_FILE_NAME = "recipients.json"
+
+
+class Exclusions:
```

Comment:
do you think it makes sense to name it CDKDeployExclusions?

constants.py is just a general-purpose file, so, maybe, name needs to be a bit more specific?

Reply:
Done